medusa/server/api/v2/search.py

```diff
@@ -0,0 +1,239 @@
+# coding=utf-8
+"""Request handler for statistics."""
+from __future__ import unicode_literals
+
+from collections import defaultdict
+from medusa import app
+from medusa.search.manual import collect_episodes_from_search_thread
+from medusa.tv.episode import Episode, EpisodeNumber
+from medusa.tv.series import Series, SeriesIdentifier
+from medusa.search.queue import (
+    BacklogQueueItem,
+    FailedQueueItem,
+    # ManualSearchQueueItem,
+    ForcedSearchQueueItem,
+)
+from medusa.server.api.v2.base import BaseRequestHandler
+
+from tornado.escape import json_decode
+
+
+class SearchHandler(BaseRequestHandler):
+    """Search queue request handler."""
+
+    #: resource name
+    name = 'search'
+    #: identifier
+    identifier = ('identifier', r'\w+')
+    #: path param
+    path_param = ('path_param', r'\w+')
+    #: allowed HTTP methods
+    allowed_methods = ('GET', 'POST')
+
+    def get(self, identifier, path_param=None, *args, **kwargs):
```

Comment:
Any reason for the `*args, **kwargs`?

Reply:
hmm yeah if I don't intend to use `path_param`. I can now use something like: `def get(self, identifier, *args, **kwargs):`

Reply:
I believe the APIv2 base handler is set up in a way that it doesn't push extra arguments.
So you could remove this - most endpoints don't have `*args, **kwargs`.

Reply:
this should be fixed